### ISSUE TITLE
How to disable Onebox?

### ISSUE THREAD

**Issue by drusepth** [thredded/thredded#693]
I very frequently get issues with Onebox throwing exceptions for threads (e.g. https://github.com/thredded/thredded/issues/678) that don't always go away when manually updating the onebox gem or thredded. Is there a way to just disable onebox support altogether?

**Comment by glebm** [comment]
Onebox shouldn't be throwing any exceptions anymore, this was fixed in #683, are you on the latest version of Thredded?

Disabling it is simple, but consider updating Thredded instead as Onebox is nice when it works:

```ruby
Thredded::ContentFormatter.after_sanitization_filters.
  delete(Thredded::HtmlPipeline::OneboxFilter)
```

**Comment by drusepth** [comment]
Oh no, I'm a little terrified of running that command, haha. 

Thanks a ton (both for the tip on disabling Onebox, but also for the actual fix). I upgraded to 4.2.10 and then upgraded and it works just fine now. You're the real MVP.
